meta/main.yml

```diff
@@ -22,4 +22,8 @@ dependencies:
   - role: geerlingguy.nodejs
     vars:
       nodejs_version: "{{ bbb_nodejs_version }}"
+    tags: geerlingguy_nodejs
```

Comment:
An additional tag `geerlingguy` would help shorten the commandline when I want to skip both dependencies.